Objective-C/Internal/Foundation+CBL.h

```diff
@@ -0,0 +1,36 @@
+//
+//  Foundation+CBL.h
+//  CouchbaseLite
+//
+//  Copyright (c) 2020 Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+#import "c4Base.h"
+#import "fleece/slice.hh"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSString (CBL)
+- (C4Slice) c4slice;
+
```

Comment:
extra empty line? What is the convention here?

Reply:
Each function will be separated with a single line, including last function. One line and then `@end`